hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/profiling/DefaultConsumerProfiler.java

```diff
@@ -0,0 +1,77 @@
+package pl.allegro.tech.hermes.consumers.consumer.profiling;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.util.StopWatch;
+import pl.allegro.tech.hermes.api.SubscriptionName;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+public class DefaultConsumerProfiler implements ConsumerProfiler {
```

Comment:
Since this class will be accessed from many threads and is mutable - can we document if it is thread safe and how it should be used with multiple threads accessing it?